app/konnect/getting-started/publish-service.md

```diff
@@ -46,27 +46,28 @@ descriptions.
 
     ```
 
-1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), open a service.
-Find the **Service Document** section on your service's overview page and click **Upload Document**.
+1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.
+
+1. Select **Documentation**, then **Add Page**. Upload the `.md` file and **Save**.
+
+    {% konnect_icon servicehub %} may automatically populate fields from the uploaded documentation.
```

Comment:
Why wouldn't it auto populate? If the fields from the file don't match the fields in Service Hub? It seems like you'd want it to automatically populate, so I'd provide info so customers can guarantee that outcome.

Reply:
good point. it always auto populates